src/lib/components/identity-card/identity-card.svelte

```diff
@@ -84,10 +87,20 @@
       <div class="icon">
         <DripListIcon style="fill: var(--color-primary); height: 3rem; width: 3rem;" />
       </div>
-      <span class="typo-header-3 ellipsis">{dripList.name}</span>
+
+      <div>
+        <span class="typo-header-3 ellipsis" class:hiddenByUser={!dripList.isVisible}
```

Comment:
Another camel case class btw

Reply:
...and some more sprinkled throughout the PR.

Reply:
👍